script/rest/update-files.js

```diff
@@ -12,12 +12,12 @@ import { program } from 'commander'
 import { execSync } from 'child_process'
 import mkdirp from 'mkdirp'
 import rimraf from 'rimraf'
-import yaml from 'js-yaml'
 
 import { decorate } from './utils/decorator.js'
+import { getSchemas } from './utils/get-openapi-schemas.js'
 
 const TEMP_DOCS_DIR = path.join(process.cwd(), 'openapiTmp')
-const DEREFERENCED_DIR = path.join(process.cwd(), 'lib/rest/static/dereferenced')
```

Comment:
> const DOCS_DEREF_OPENAPI_DIR = path.join(process.cwd(), 'lib/rest/static/dereferenced')
 const GITHUB_REP_DIR = path.join(process.cwd(), '../github')
 const OPEN_API_RELEASES_DIR = path.join(GITHUB_REP_DIR, '/app/api/description/config/releases')

program
  .description('Generate dereferenced OpenAPI and decorated schema files.')
  .option(
    '--decorate-only',
    '⚠️ Only used by a 🤖 to generate decorated schema files from existing dereferenced schema files.'
  )
  .option(
    '-v --versions <VERSIONS...>',
    'A list of undeprecated, published versions to build, separated by a space. Example "ghes-3.1" or "api.github.com github.ae"'
  )
  .option('-d --include-deprecated', 'Includes schemas that are marked as `deprecated: true`')
  .option('-u --include-unpublished', 'Includes schemas that are marked as `published: false`')
   .parse(process.argv)

 const { decorateOnly, versions, includeUnpublished, includeDeprecated } = program.opts()
 const versionsArray = versions ? versions.split(' ') : []

 await validateInputParameters()

main()
async function main() {
   // When the input parameter type is decorate-only, use the
   // `github/docs-internal` repo to generate a list of schema files.
   // Otherwise, use the `github/github` list of config files